### PR TITLE
completed prep-javascript-objects

### PR DESCRIPTION
**Pull request by Smith-Steve**

### CODE REVIEW

**Comment by Smith-Steve** [comment]
image

**Comment by Smith-Steve** [comment]
image

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+console.log('The person\'s full name is: ' + fullName);
+console.log(fullName + '\'s job is to be a : ' + person.job);
+console.log(fullName + '\'s previous job was a: ' + person.previousJob);
+console.log('This is a console log of the entire person object. Let\'s see if it worked!');
+console.log(person);
+
+var myCar = { make: 'Chrysler', model: 'GT Lebaron', year: 1986, interior: 'Cheetah-Zebra' };
+console.log(myCar);
+
+myCar['cwner'] = fullName;
```
Comment (thebearingedge) [inline prep-javascript-objects/main.js]: This should be `myCar['owner']` with an `o`. 😄

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great! Just one small change.

**Review by thebearingedge — APPROVED** [review]
Approved!

**Pull request merged**